prac_06/guitars.py

```diff
@@ -0,0 +1,24 @@
+"""
+Guitar playing program
+Estimate: 15 minutes
+Actual:   10 minutes
+"""
+
+from guitar import Guitar
+
```

Comment:
You could also refactor this could into a function, or two functions

Reply:
👍